.github/workflows/unit-tests.yaml

```diff
@@ -101,12 +101,12 @@ jobs:
                     extensions: ctype, iconv, xml
 
             -   name: Install Composer dependencies
-                uses: ramsey/composer-install@v1
+                uses: ramsey/composer-install@v2
 
             -   name: Install RequirementChecker Composer dependencies
-                uses: ramsey/composer-install@v1
+                uses: ramsey/composer-install@v2
                 with:
-                    working-directory: 'vendor-bin/requirement-checker'
+                    working-directory: 'requirement-checker'
```

Comment:
🤦

Reply:
Darn! I still missed one ?

Reply:
no no, I just realised the mistake I did there 😄

Reply:
Ah! Well, the good thing about the new version of the action is that it was failing because of it, so no chance of it happening again 😁